tools/bin/build_report.py

```diff
@@ -30,6 +30,7 @@
 DESTINATION_DEFINITIONS_YAML = f"{CONNECTOR_DEFINITIONS_DIR}/destination_definitions.yaml"
 CONNECTORS_ROOT_PATH = "./airbyte-integrations/connectors"
 RELEVANT_BASE_MODULES = ["base-normalization", "source-acceptance-test"]
+CONNECTOR_BUILD_OUTPUT_URL = "https://dnsgjos7lj2fu.cloudfront.net/tests/history/connectors"
```

Comment:
@sherifnada @evantahler I just wanted to make you aware of this path change. (And the related changes to `builds.md` and `report.sh`).

Essentially we had two paths that were in use for connector builds status
1. `tests/history`
2. `tests/history/connectors`

This change wrangles this to ensure we are always using and referencing `tests/history/connectors`

Let me know if theres a system impacted that I may have missed